### ISSUE TITLE
≢ parser bug

### ISSUE THREAD

**Issue by cscherrer** [JuliaLang/julia#42615]
Thanks to @Seelengrab for helping me track this down:
```julia
julia> 2 ≢ 3
true

julia> 2 ≢ 3
true

julia> "≢" ≢ "≢"
true
```

On some displays, these render the same. And both work, sometimes. But only one of them works in CI, see
https://github.com/cscherrer/MeasureTheory.jl/runs/3875292043?check_suite_focus=true

```julia
julia> versioninfo()
Julia Version 1.7.0-rc1
Commit 9eade6195e (2021-09-12 06:45 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: AMD Ryzen Threadripper 2950X 16-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, znver1)
Environment:
  JULIA_NUM_THREADS = 32
  JULIA_EDITOR = code
```

**Comment by brenhinkeller** [comment]
Looks fine on 1.8.3 and 1.9.0-alpha1; not sure if we have to wait to close it though given 1.6 is still LTS

**Comment by GunnarFarneback** [comment]
I don't remember what fixed what but at least #41870 was backported to 1.6.

**Comment by brenhinkeller** [comment]
Seems like we can call it fixed then, but if anyone disagrees please reopen!
